packages/server/lib/config.coffee

```diff
@@ -16,8 +16,32 @@ pathHelpers = require("./util/path_helpers")
 cypressEnvRe = /^(cypress_)/i
 dashesOrUnderscoresRe = /^(_-)+/
 
-folders = "fileServerFolder videosFolder supportFolder fixturesFolder integrationFolder screenshotsFolder unitFolder supportFile".split(" ")
-configKeys = "port reporter reporterOptions baseUrl execTimeout defaultCommandTimeout pageLoadTimeout requestTimeout responseTimeout numTestsKeptInMemory screenshotOnHeadlessFailure waitForAnimations animationDistanceThreshold watchForFileChanges trashAssetsBeforeHeadlessRuns chromeWebSecurity videoRecording videoCompression videoUploadOnPasses viewportWidth viewportHeight supportFile fileServerFolder supportFolder fixturesFolder integrationFolder videosFolder screenshotsFolder environmentVariables hosts".split(" ")
+folders = """
+  fileServerFolder   fixturesFolder   integrationFolder   screenshotsFolder
+  supportFile        supportFolder    unitFolder          videosFolder
+""".trim().split(/\s+/)
+
+configKeys = """
+  animationDistanceThreshold      fileServerFolder
+  baseUrl                         fixturesFolder
+  chromeWebSecurity               integrationFolder
+  environmentVariables            pluginsFile
+  hosts                           screenshotsFolder
+  numTestsKeptInMemory            supportFile
+  port                            supportFolder
+  reporter                        videosFolder
+  reporterOptions
+  screenshotOnHeadlessFailure     defaultCommandTimeout
+  testFiles                       execTimeout
+  trashAssetsBeforeHeadlessRuns   pageLoadTimeout
+  viewportWidth                   requestTimeout
+  viewportHeight                  responseTimeout
+  videoRecording
+  videoCompression
+  videoUploadOnPasses
+  watchForFileChanges
+  waitForAnimations
+""".trim().split(/\s+/)
```

Comment:
pretty good idea to have text block here. Maybe separate `.trim().split(/\s+/)` to named function that describes its purpose like `toWords`